c10/core/TensorImpl.h

```diff
@@ -586,6 +586,25 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
    */
   const at::Tensor& grad() const;
 
+
+  /**
+   * Whether or not the imaginary part of the tensor should be conjugated
```

Comment:
Technically, you only conjugate a complex number; conjugation involves negating the imaginary part of the tensor.